src/command_modules/azure-cli-network/azure/cli/command_modules/network/zone_file/make_zone_file.py

```diff
@@ -54,32 +46,58 @@ def make_zone_file(json_zone_file, template=None):
         "uri":     [ uri records ]
     }
     """
+    import sys
+    import azure.cli.command_modules.network.zone_file.record_processors as record_processors
+    from six import StringIO
+
+    old_stdout = sys.stdout
```

Comment:
Any specific reason we need to hide the sys.stdout? will `zone_file.write` not sufficient? My concern is on unknown exceptions, we might not have chance to restore the sys.stdout.